masterlist.yaml

```
@@ -21511,6 +21511,91 @@ plugins:
   - name: 'Holds - Falkreath.esp'
     after: [ 'Landscape Fixes For Grass Mods.esp' ]
 
+  - name: HSDragonBridge - Four Shields Tavern.esp
```

It'd also help to add a subsection header to help clarify why these mods are all clustered together. If you already have an official name for this series, you could use that. If not, you could use something like this:

```suggestion
  ### HS Interior Locations ###
  - name: HSDragonBridge - Four Shields Tavern.esp
```